horizons/util/python/decorators.py

```diff
@@ -97,12 +97,11 @@ def __call__(self, *args, **kwargs):
 					del self.cache_dates[key]
 					return self(*args, **kwargs)
 			else:
-				self.cache_dates[key] = time.time() # new entry
+				self.cache_dates[key] = time.time()  # new entry
```

Comment:
Thank you for find the old statement from @ChrisOelmueller. I think I make it better for code reading with a good example, but this end with

> I don't see the point in arguing about this longer.

In my opinion, we should talk about it a bit. Even if it is annoying for you. *joke
I not ignoring that statement, in my first answer was the are two other line the agrees with the pep8,  in my first was make the file consistent.

> rest of the codebase to see that (apparantly) most inline comments use one space

I think logical: consistent = all the same = true,
most all the same = not consistent = false